libs/shared/ui/src/lib/components/table/table-row-deployment/table-row-deployment.tsx

```diff
@@ -149,9 +145,22 @@ export function TableRowDeployment(props: TableRowDeploymentProps) {
                   {timeAgo(data?.updated_at ? new Date(data?.updated_at) : new Date(data?.created_at || ''))} ago
                 </span>
               </p>
-              {index === 0 && data?.name && !(data as ContainerApplicationEntity)?.image_name && (
-                <ButtonIconAction actions={buttonActionsDefault} />
-              )}
+              <ButtonIcon
+                dataTestId="btn-logs"
+                icon={IconAwesomeEnum.SCROLL}
+                style={ButtonIconStyle.STROKED}
+                size={ButtonSize.SMALL}
+                onClick={() =>
+                  navigate(
+                    fromService
+                      ? pathEnvironmentLogs + SERVICE_LOGS_URL(data?.id)
+                      : pathEnvironmentLogs +
+                          DEPLOYMENT_LOGS_VERSION_URL(data?.id, (data as DeploymentService).execution_id)
+                  )
+                }
+                className="!w-7 !h-7 !border-r btn-icon-action__element"
```

Comment:
Add important is a bad practice. We must find another way to style stuff without overriding existing rules